### DISCUSSION TITLE
number of shards

### DISCUSSION THREAD

**Discussion by trazhub** [jagrosh/MusicBot#1297] [Questions]
Hey jagrosh how can change the
 number of shards i want to make this use multiple shards how can i do that ?
![image](https://user-images.githubusercontent.com/107756312/220611431-fb540acf-2cd8-4dc4-8b7e-bfc915f76f17.png)

**Comment by jagrosh** [comment]
There is no reason to run JMusicBot on multiple shards; you'll only encounter worse performance by the additional threads and memory needed, and no benefits, as JMusicBot is intended to be run as a private bot on fewer than 75 servers.
